controllers/substitute.js

```diff
@@ -148,6 +153,10 @@ function saveToDB(passon) {
         metadataToSave = passon.baseMetaData;
         metadataToSave.substituted = passon.metadata.substituted;
         metadataToSave.substitution = passon.metadata.substitution;
+        // change title of Substituted ERC
+        metadataToSave.raw.title = 'Subst && ' + metadataToSave.raw.title;
```

Comment:
Please do _not_ change the title. As agreed in the API spec, just put in `substituted: true`.